content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Agent Transport for logs
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
+
+Since Agent v6.19+/v7.19+, the default transport is HTTPS with compression instead of TCP.
+At agent start, if log collection is enabled, the agent will run a HTTPS connectivity test:
+
+* if successfull, then the agent will use HTTPS transport
+* otherwise the agent will fall back to TCP transport
+
+This connectivity test mechanism is only running at agent startup. If the connectivity is failing for both HTTPS and TCP at agent startup time, then the agent will use TCP transport until its next restart (and then rerun a new Connectivity test)
+
+To check which transport is used by the agent, run the [agent statuts command][1].
```

Comment:
Let's put a screenshot of what it can look like

Reply:
added